packages/commonwealth/client/scripts/views/Sublayout.tsx

```diff
@@ -22,6 +23,12 @@ const Sublayout = ({
   hasCommunitySidebar,
 }: SublayoutProps) => {
   const forceRerender = useForceRerender();
+  const [toggleMobileView, setToggleMobileView] = useState(
+    (location.pathname.includes('discussions') ||
+      location.pathname.includes('search')) &&
+      window.innerWidth <= 425
```

Comment:
why 425 exactly? Also, we have bunch of utils that handles the breakpoints and check innerWidth of the window, especially, that it is enclosed in nice hook `useBrowserWindow`. We should not add custom code that repeats the functionality of created utils

Reply:
This will be addressed in the newest commit.